website/common/locales/en/settings.json

```diff
@@ -62,8 +63,8 @@
     "newUsername": "New Login Name",
     "dangerZone": "Danger Zone",
     "resetText1": "WARNING! This resets many parts of your account. This is highly discouraged, but some people find it useful in the beginning after playing with the site for a short time.",
-    "resetText2": "You will lose all your levels, gold, and experience points. All your tasks (except those from challenges) will be deleted permanently and you will lose all of their historical data. You will lose all your equipment but you will be able to buy it all back, including all limited edition equipment or subscriber Mystery items that you already own (you will need to be in the correct class to re-buy class-specific gear). You will keep your current class and your pets and mounts. You might prefer to use an Orb of Rebirth instead, which is a much safer option and which will preserve your tasks and equipment.",
-    "deleteLocalAccountText": "Are you sure? This will delete your account forever, and it can never be restored! You will need to register a new account to use Habitica again. Banked or spent Gems will not be refunded. If you're absolutely certain, type your password into the text box below.",
+    "resetText2": "You will lose all your levels, gold, and experience points. All your tasks (except those from challenges) will be deleted permanently and you will lose all of their historical data. You will lose all your equipment but you will be able to buy it all back, including all limited edition equipment or subscriber Mystery items that you already own (you will need to be in the correct class to re-buy class-specific gear). You will keep your current class and your pets and mounts. You might prefer to use an Orb of Rebirth instead, which is a much safer option and which will preserve your tasks.",
+    "deleteLocalAccountText": "<b>Are you sure? This will delete your account forever, and it can never be restored! You will need to register a new account to use Habitica again. Banked or spent Gems will not be refunded. If you're absolutely certain, type your password into the text box below.</b>",
```

Comment:
@JSchneidler Can you please add "and equipment" back on to the end of `resetText2`? The Orb of Rebirth no longer removes gear.

For `deleteLocalAccountText`, can you do the bolding in the Jade file that uses that string instead of having a tag in the locales files? That will avoid any possibility of the translators modifying the bold tag, and also means we can change the formatting later without needing to duplicate the change in all the language files.

How are you going with this PR? Do you have any questions about any of it or need any advice?